contrib/packs/actions/workflows/deploy.yaml

```diff
@@ -26,7 +26,23 @@ workflows:
 
         on-complete:
           - do_auto_install: <% $.auto_deploy = true %>
-          - do_manual_install: <% $.auto_deploy = false %>
+          - check_for_no_subtree: <% $.auto_deploy = false %>
+
+      check_for_no_subtree:
+        action: core.noop
+
+        on-complete:
+          - do_manual_install: <% len($.packs) != 0 %>
+          - publish_packs: <% len($.packs) = 0 %>
+
+      publish_packs:
+        action: core.noop
+
+        publish:
+          packs: [ <% $.repo_name %> ]
```

Comment:
Ah, OK, that makes sense. Thanks for the clarification.

It would also be good to add a comment to the workflow which clarifies that :)